frontend/src/cards/MapCard.tsx

```diff
@@ -538,10 +545,10 @@ function MapCardWithKey(props: MapCardProps) {
                       highestLowestGroupsByFips={highestLowestGroupsByFips}
                       activeDemographicGroup={activeDemographicGroup}
                       countColsMap={countColsMap}
-                      data={displayData}
+                      data={displayData as DataPoint[]}
                       filename={filename}
                       fips={props.fips}
-                      geoData={geoData}
+                      geoData={geoData as Topology}
```

Comment:
is it not possible to cast variables as the correct types when the variables are created? Rather than creating them and then using `as` when we need to send them into the component?